### PR TITLE
feat(integrations): Add troubleshooting docs for Vercel, JIRA and Azure DevOps

### PR DESCRIPTION
**Pull request by manuzope**
Some recent issues that have come up because of our being on easylist's blocked list and an Azure Devops issue that has a workaround.

JIRA issue: https://github.com/getsentry/sentry/issues/24412
Vercel issue: https://github.com/getsentry/sentry/issues/23284#issuecomment-778181343
Azure Devops issue: https://github.com/getsentry/sentry/issues/23689

### CODE REVIEW

File: src/docs/product/integrations/azure-devops/index.mdx [hunk 1]
```diff
@@ -111,3 +111,4 @@ If you reach the account selection page during the Azure Devops installation pro
 - Ensure you are logged into the account connected to your Azure DevOps organization.
 - Double-check that your account is a Microsoft Account (MSA).
 - For Single Sign-On, see [Azure Active Directory SSO](/product/accounts/sso/azure-sso/).
+- If you have multiple accounts in Azure DevOps, open https://aex.dev.azure.com in another tab, select the correct account and redo the installation. If that works, please upvote https://github.com/getsentry/sentry/issues/14357, which is where we're tracking this improvement.
```
Comment (PeloWriter) [inline src/docs/product/integrations/azure-devops/index.mdx:114]:
```suggestion
- Ensure you are logged into the account connected to your Azure DevOps organization.
- Double-check that your account is a Microsoft Account (MSA).
- For Single Sign-On, see [Azure Active Directory SSO](/product/accounts/sso/azure-sso/).
- If you have multiple accounts in Azure DevOps, open [this link to Azure DevOps](https://aex.dev.azure.com) in another tab, select the correct account, then reinstall.

 <Note>
 
 If reinstallation solves the problem, please upvote [that solution in GitHub](https://github.com/getsentry/sentry/issues/14357), which is where we're tracking this improvement.
 
 </Note>
```

File: src/docs/product/integrations/jira/index.mdx [hunk 2]
```diff
@@ -152,8 +152,16 @@ Alert notifications in Sentry can be routed to many supported integrations, but
 
 ## Troubleshooting
 
+### Trouble with on premise installation
```
Comment (PeloWriter) [inline src/docs/product/integrations/jira/index.mdx:155]:
```suggestion
### Trouble with Self-Hosted Sentry Installation
```

File: src/docs/product/integrations/jira/index.mdx [hunk 3]
```diff
@@ -152,8 +152,16 @@ Alert notifications in Sentry can be routed to many supported integrations, but
...
+
 If you're having trouble setting up Sentry with your on-premises integration, verify the following:
```
Comment (PeloWriter) [inline src/docs/product/integrations/jira/index.mdx:157]:
```suggestion
If you're having trouble setting up Self-Hosted Sentry, verify the following:
```

File: src/docs/product/integrations/jira/index.mdx [hunk 4]
```diff
@@ -152,8 +152,16 @@ Alert notifications in Sentry can be routed to many supported integrations, but
...
 
 - The provided installation URL is a fully qualified domain name (FQDN), which is resolvable on the internet.
 - The IP addresses Sentry uses to make outbound requests are [allowed](/product/security/ip-ranges/).
 - Sentry's access to your installation URL is not path restricted.
+
+### "Something went wrong" message when installing JIRA on SaaS sentry.io
```
Comment (PeloWriter) [inline src/docs/product/integrations/jira/index.mdx:163]:
```suggestion
### "Something went wrong" Message with JIRA on SaaS sentry.io Install
```

File: src/docs/product/integrations/vercel/index.mdx [hunk 5]
```diff
@@ -67,3 +67,11 @@ Use Vercel to [link projects](#project-linking) for uploading source maps and no
...
 
    ![Sentry modal showing uninstalling Vercel integration](vercel_delete_internal_integration.png)
+
+## Troubleshooting
+
+### Failed to fetch
+
+![](vercel_failed_to_fetch.png)
+
+This usually happens if you have an ad blocker that is blocking the conversation between Vercel and Sentry during setup. To remediate this, disable your ad blocker and go through the installation flow again.
```
Comment (PeloWriter) [inline src/docs/product/integrations/vercel/index.mdx:77]:
```suggestion
This issue typically occurs if you have an ad blocker blocking the conversation between Vercel and Sentry during setup. To remediate the issue, disable your ad blocker and go through the installation flow again.
```

File: src/docs/product/integrations/jira/index.mdx [hunk 6]
```diff
@@ -152,8 +152,16 @@ Alert notifications in Sentry can be routed to many supported integrations, but
...
+
+![](jira-something-went-wrong.png)
+
+This usually happens if you have an ad blocker that is blocking the conversation between JIRA and Sentry during setup. To remediate this, disable your ad blocker and go through the installation flow again.
```
Comment (PeloWriter) [inline src/docs/product/integrations/jira/index.mdx:167]:
```suggestion
This issue typically occurs if you have an ad blocker blocking the conversation between JIRA and Sentry during setup. To remediate this, disable your ad blocker and go through the installation flow again.
```

**Review by PeloWriter — APPROVED** [review]
Minor smoothing. I also switched on-premises to self-hosted as that's the updated name.

**Comment by manuzope** [comment]
Tagging @getsentry/ecosystem for visibility. With the troubleshooting section, hoping we can keep repeated or known issues to a minimum.

**Pull request merged**